rllib/env/normalize_actions.py

```diff
@@ -0,0 +1,20 @@
+import gym

```

Comment:
This wrapper has recently been added in the gym master. Not sure if we're able upgrade gym version for rllib, but if we are, I think we should use the standard implementation.

https://github.com/openai/gym/blob/master/gym/wrappers/rescale_action.py

Reply:
I think that's fine yeah, we can raise an error telling the user to upgrade if it's not found.